app/pages/task-list/existingorder/ui-tests/ui.test.js

```diff
@@ -555,6 +555,210 @@ test('should only render the complete tag for task 6 item 1 when returned as com
     .expect(task6Item1CompleteTag.exists).ok();
 });
 
+
+test('should render task 7 item 1', async (t) => {
+  await pageSetup(t, true);
+  await t.navigateTo(pageUrl);
+
+  const task7 = Selector('li[data-test-id="task-6"]');
+  const task7Item1 = Selector('li[data-test-id="task-6-item-0"]');
+
+  await t
+    .expect(task7.exists).ok()
+    .expect(await extractInnerText(task7.find('h2 span'))).eql('7.')
+    .expect(await extractInnerText(task7.find('h2 div'))).eql('Add Associated Services')
+    .expect(task7Item1.exists).ok()
+    .expect(await extractInnerText(task7Item1.find('span'))).eql('Add Associated Services to your order');
+});
+
+test('should render task 7 item 1 as text if recipients saved and count 1', async (t) => {
+  const mockOrderSummary = generateMockOrderSummary([
+    { id: 'ordering-party', status: 'complete' },
+    { id: 'supplier', status: 'complete' },
+    { id: 'commencement-date', status: 'complete' },
+    { id: 'service-recipients', status: 'complete', count: 1 },
+  ]);
+  await pageSetup(t, true, mockOrderSummary);
+  await t.navigateTo(pageUrl);
+
+  const task7Item1 = Selector('li[data-test-id="task-6-item-0"]');
+
+  await t
+    .expect(task7Item1.find('a').exists).notOk();
+});
+
+test('should only render task 7 item 1 as a link if recipients saved and count 0', async (t) => {
+  const mockOrderSummary = generateMockOrderSummary([
+    { id: 'ordering-party', status: 'complete' },
+    { id: 'supplier', status: 'complete' },
+    { id: 'commencement-date', status: 'complete' },
+    { id: 'service-recipients', status: 'complete', count: 0 },
+  ]);
+  await pageSetup(t, true, mockOrderSummary);
+  await t.navigateTo(pageUrl);
+
+  const task7Item1 = Selector('li[data-test-id="task-6-item-0"]');
+
+  await t
+    .expect(task7Item1.find('a').exists).ok()
+    .click(task7Item1.find('a'))
+    .expect(getLocation()).eql(`http://localhost:1234${baseUrl}/organisation/order-id/associated-services`);
+});
+
+test('should only render task 7 item 1 as a link if recipients saved and count 1, Catalogue solution saved and count 0', async (t) => {
+  const mockOrderSummary = generateMockOrderSummary([
+    { id: 'ordering-party', status: 'complete' },
+    { id: 'supplier', status: 'complete' },
+    { id: 'commencement-date', status: 'complete' },
+    { id: 'service-recipients', status: 'complete', count: 1 },
+    { id: 'catalogue-solutions', status: 'complete', count: 0 },
```

Comment:
Does it not need the additional service to also be complete for it to be a link?

Reply:
not if catalogue solutions is complete with 0 added it's scenario 3